code/game/machinery/computer/medical.dm

```diff
@@ -128,6 +138,17 @@
 	onclose(user, "med_rec")
 	return
 
+/obj/machinery/computer/med_data/proc/pathogen_dat(var/datum/data/record/v)
```

Comment:
This block looks like it's commented out.

Reply:
Yeah I've replaced the ability to change a disease's name in the database with the ability to give it a nickname, so the ID still properly appears. I'm keeping the old name change lines commented out in case it becomes an issue for some reason, but I might just remove it later.